src/components/menu/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { styled } from '@linaria/react'
+import { Text2XS } from '../typography'
+
+export const ElMenuPopover = styled.div`
+  position: absolute;
+  white-space: nowrap;
+  z-index: 98;
+`
+
+export const ElMenu = styled.div`
+  overflow: hidden;
+  display: flex;
+  min-width: 200px;
+  width: fit-content;
+  padding: var(--spacing-2) var(--spacing-none);
+  border-radius: var(--corner-default);
+  background-color: var(--fill-white);
+  box-shadow: 0px 4px 16px 0px #222b3329;
+`
+
+export const ElMenuItemContainer = styled.li`
```

Comment:
initially its because its a `li` element, which later the consumer use it to wrap their list item e.g: button, link, or .etc. and if we used without any props it will not display anything

Reply:
but now I think `ElMenuItem` is simpler and still intuitive